backend/Builtins.icl

```diff
@@ -463,6 +464,16 @@ bs_module = (["module", "definition", "implementation", "system", "definition mo
 		]
 	})
 
+bs_pattern_named = (["=:"],
+	{ syntax_title         = "newtype definition (experimental)"
+	, syntax_code          = [":: ... =: T c"]
+	, syntax_description   = "Define a newtype, a newtype is a type synonym that is treated as a real type."
```

Comment:
- Comma splice; the first three words can be removed.
- I think the wording is a little confusing because it is not clear that the type synonym is the run-time interpretation and the real type is the compile-time interpretation. Perhaps "A newtype is a type with one constructor which exists during type checking, but not at run-time." or so.
- You could add a second sentence with why this is useful (defining other instances without run-time overhead).